web/js/validation_rules.js

```diff
@@ -1,5 +1,11 @@
     validation_rules = {
         title: { required: true },
         detail: { required: true },
-        update: { required: true }
+        update: { required: true },
+        password_register: {
+          remote: {
+            url: '/auth/common_password',
+            type: 'post'
```

Comment:
Why do only the common password bit here but not the length?

Reply:
I was going to say the length is handled by fixmystreet.js in the addMethod, but that's then fixed whereas I made server-side cobrand specific didn't I, hmm.

Reply:
Will still do it client side as it's quicker, but will pass a variable from server to client.